### PR TITLE
reindexing: use streaming_bulk action

### PR DESCRIPTION
**Pull request by MJedr**

### CODE REVIEW

File: backend/inspirehep/indexer/base.py [hunk 1]
```diff
@@ -99,11 +99,21 @@ def bulk_index(self, records_uuids, request_timeout=None, max_chunk_bytes=None):
             initial_backoff=10,  # wait for initial_backoff * 2^retry_number,
             max_chunk_bytes=max_chunk_bytes,
         )
+        nb_of_success = 0
+        nb_of_failures = 0
+        failures = []
+        for item in result:
```
Comment (drjova) [inline backend/inspirehep/indexer/base.py:105]:
```suggestion
        for action_success, action_data in result:
```
🙃

File: backend/inspirehep/indexer/base.py [hunk 2]
```diff
@@ -99,11 +99,21 @@ def bulk_index(self, records_uuids, request_timeout=None, max_chunk_bytes=None):
...
+            # each item is a tuple of (action_success, action_data)
+            if not item[0]:
+                failures.append(item[1]["index"])
+                nb_of_failures += 1
```
Comment (drjova) [inline backend/inspirehep/indexer/base.py:109]: can we just len(failures) at the end?

File: backend/inspirehep/indexer/base.py [hunk 3]
```diff
@@ -100,10 +100,15 @@ def bulk_index(self, records_uuids, request_timeout=None, max_chunk_bytes=None):
...
+        for action_success, action_data in result:
+            if not action_success:
+                failures.append(action_data["index"])
+
         return {
-            "success": success,
+            "success_count": len(records_uuids) - len(failures),
+            "failures_count": len(failures),
             "failures": failures,
-            "failures_count": len(records_uuids) - success,
```
Comment (drjova) [inline backend/inspirehep/indexer/base.py:106]: tiny: we can store `len(failures)` in a var and not calculate it twice

**Review by drjova — APPROVED** [review]
LGTM just the comment

File: backend/inspirehep/indexer/base.py (hunk 3 above)
Comment (MJedr) [reply backend/inspirehep/indexer/base.py:106]: yes, here you are

**Comment by drjova** [comment]
@MJedr :shipit:

**Pull request merged**